docs/guide/docs/main/overview.md

```diff
@@ -1,112 +1,154 @@
 
-## Dumb vs Smart
+### Rule-Based Chatbots
+A chatbot can be **rule-based** meaning it doesn't understand much about conversations. Such a chatbot usually relies heavily on pre-populated responses (e.g., buttons) or one-word answers. User messages rarely influence responses. The chatbot moves to the next step after receiving a user message or prompts the user to give a pre-configured reply (e.g., a button or keyword).
 
-A bot is simply a computer program that uses conversations as its main user interface. We call these interfaces **Conversational User Interfaces** (CUI). A bot can be **dumb** (don't understand much about conversations) or **smart** (understand language and communicate in a human-like way).
+This type of chatbot is cheaper, easy to implement, 99% effective, and solves issues fast. It has these characteristics because the developer built the chatbot for a specific workflow. Once a user understands the chatbot's goal, s/he can easily use the chatbot for this goal.
 
-While you can develop both kind of bots with Botpress, you will really benefit most of it if you're trying to build a smart bot.
+### AI-Based Chatbots
+These chatbots can understand language and communicate in a human-like way. They can extract useful information from conversations like dates, amounts, and locations.
 
-## Anatomy of a bot
+These chatbots are indispensable when you need to harvest all available data from the user's message, like dates, quantities, user's mood, and amounts. They can also help streamline conversations by gathering as much information as possible from user responses, making conversations shorter and more concise.
 
-A bot is pretty simple:
+While you can use Botpress to develop both kinds of chatbots, you will benefit most from the framework if you're trying to build an A.I.-based chatbot.
 
-1. It receives messages from Messaging **Channels**
-2. It **processes** these messages to understand, translate or escalate them
-3. It **decides** on what to respond back to the user
+## Anatomy of a chatbot
+A chatbot is quite simple:
 
-#### Lifecycle
+1. It receives messages from Messaging **channels**.
+2. It **processes** these messages to understand, translate, or escalate underlying information.
+3. It **decides** how to respond to the user
+![High-Level Diagram](../assets/hld.png)
+The red dots are the developer extension points. These indicate where a chatbot developer can inject code to customize their chatbot.
 
-![High-Level Diagram](assets/hld.png)
+### Channels
+Think of a channel in the context of communication. Using this definition, we can describe a channel as any chat platform which connects your chatbot to the intended user, for example, Facebook Messenger. Botpress provides pre-built modules which allow your chatbot to send and receive messages from the following chat platforms:
+ 
+ - Website Embedded Chat
```

Comment:
How about HTTP API (converse) do we count this as a channel ? @marcmerce @Michael-N-M

Reply:
Let's keep it as is. Converse is more than a channel, some users use it as a channel but some other use it to integrate to their own channel. Moreover, messaging api is coming so that'll become super confusing.